src/components/LoanCards/HoverLoanCard/HoverLoanCard.vue

```diff
@@ -170,12 +177,22 @@ export default {
 		},
 		handleClick() {
 			if (!this.hoverEffectActive()) {
-				this.$emit('update-detailed-loan-index', this.loanIndex);
+				this.updateDetailedLoanIndex();
 			}
 		},
 		trackInteraction(args) {
 			this.$emit('track-interaction', args);
 		},
+		hoverCardLargeUpdateDetailedLoanIndex() {
+			if (!this.rowHasDetailedLoan) {
+				this.$emit('set-prevent-updating-detailed-card', true);
```

Comment:
1) User clicks on Image or Read more inside `HoverLoanCardLarge`
  * This stops the mouseenter behavior of changing which loan is detailed